lib/eex/mix.exs

```diff
@@ -2,10 +2,6 @@ defmodule EEx.MixProject do
   use Mix.Project
 
   def project do
-    [
-      app: :eex,
-      version: System.version(),
-      build_per_environment: false
-    ]
+    [app: :eex, version: System.version(), build_per_environment: false]
```

Comment:
This is bad, the project definition in the mix.exs file should always be multiple lines.

Reply:
Indeed, I'd personally have every list-wrapped keyword pairs be multi-line.  If not list wrapped (say in a function call) then it could try to collapse it if possible.